content/en/dashboards/_index.md

```diff
@@ -96,19 +99,37 @@ An organization's most popular dashboard displays five popularity bars. All othe
 
 ## New dashboard
 
-To create a dashboard, click on the **New Dashboard +** button in the upper right. Enter a dashboard name and choose between creating a [timeboard][3] or a [screenboard][4].
+<div class="alert alert-warning">The new dashboard functionality is in public beta.</div>
+
+To create a dashboard, click on the **New Dashboard +** button in the upper right. Enter a dashboard name and choose a layout option.
+
+{{< img src="dashboards/create-dashboard.jpeg" alt="Adding a new dashboard"  style="width:70%;">}}
+
+### Dashboard layout
+
+<div class="alert alert-warning">The dashboard layout functionality is in public beta.</div>
```

Comment:
> The Dashboard layout is in public beta. Screenboards and Timeboards remain the same.

- capitalizing Dashboard to mark it as a specific new feature (aka multi-size layout) 
- adding the screenboard/timeboard caveat so it's clear users can keep their old dashboards